v22.2/ui-schedules-page.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Schedules Page
+summary: The Schedules page of the DB Console provides details about scheduled tasks performed by your cluster.
+toc: true
+docs_area: reference.db_console
+---
+
+{% include_cached new-in.html version="v22.2" %} The **Schedules** page of the DB Console provides details about the scheduled tasks performed by your cluster. These can include:
+
+- [Scheduled backups](manage-a-backup-schedule.html)
+- [Scheduled auto statistics compaction](cost-based-optimizer.html#table-statistics)
+- [Row-level TTL](row-level-ttl.html)
+
+To view these details, [access the DB console](ui-overview.html#db-console-access) and click **Schedules** in the left-hand navigation.
+
+## Filter schedules
+
+- Use the **Status** menu to filter schedules by schedule status.
```

Comment:
nit: let's provide the available options to select from, like we do for the **Show** menu just beneath:

> Use the **Status** menu to filter schedules by status, displaying all schedules, only active schedules, or only paused schedules.